scripts/tools/check_blocker_staleness.py

```
@@ -0,0 +1,335 @@
+    issue: IssueRef
+    blocker_numbers: tuple[int, ...]
+    closed_blockers: tuple[int, ...]
+    open_blockers: tuple[int, ...]
+    unknown_blockers: tuple[int, ...]
+    classification: BlockerClassification
+
+
+def _normalize_heading(title: str) -> str:
+    """Normalize a Markdown heading for blocker-section matching."""
+
+    cleaned = re.sub(r"[:：]+$", "", title.strip())
```

Ruff (0.15.17)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)